stacks/api/bucket/bucket-data.controller.ts

```
@@ -456,6 +475,18 @@ export class BucketDataController {
       .then(result => result.deletedCount);
     if (deletedCount < 1) return;
 
+    if (this.history) {
+      await Promise.all(
+        body.map(id => {
+          this.history
+            .deleteMany({
+              document_id: new ObjectId(id)
+            })
+            .catch(err => err);
```

Why do we ignore the error here?